conditions/README.md

```diff
@@ -0,0 +1,341 @@
+
+## Unmapped fields
+
+- implicitRules
+- language
+- text
+- contained
+- extension
+- modifierExtension
+- identifier.use
+- identifier.type
+- identifier.type.coding
+- active
+- name
+- gender
+- birthDate
+- deceasedDateTime
+- address
+- maritalStatus
+- multipleBirthBoolean
+- generalPractitioner
+- link
+
+
+## JSON FHIR > XML HL7
+
+A FHIR Condition is mapped to a HL7 LinkSet and an associated ObservationStatement
+
+| Mapped to (XML HL7 LinkSet)                     | Mapped from (JSON FHIR / other source )                                                                                                                                                                                        |
+|-------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| id [@root]                                      | Unique ID generated by the Adaptor                                                                                                                                                                                             |
+| code [@code]                                    | if `Observation.clinalStatus` equals `"active"` then `"394774009"` (Active problem) else `"394775005"` (Inactive problem)                                                                                                      |
+| code [@displayName]                             | if `Observation.clinalStatus` equals `"active"` then `"Active Problem"` else `"Inactive Problem"`                                                                                                                              |
+| code / originalText                             | Same as `code [@displayName]` and appended with `Condition.extension[index].value` <sup>4</sup>                                                                                                                                |
+| code / qualifier [@inverted]                    | fixed value = `"false"`                                                                                                                                                                                                        |
+| code / qualifier / name [@code]                 | if `Condition.extension[index].value` <sup>4</sup> equals `"Major"` then `"386134007"` else if equals `"Minor"` then `"394847000"`                                                                                             |
+| code / qualifier / name [@codeSystem]           | fixed value = `"2.16.840.1.113883.2.1.3.2.4.15"`                                                                                                                                                                               |
```

Comment:
Worth mentioning we also set this fixed value on `code [@codeSystem]` too?

Reply:
Yes, I'll add it